pkg/linter/linter.go

```diff
@@ -152,10 +152,38 @@ func New(cxt *portercontext.Context, mixins pkgmgmt.PackageManager) *Linter {
 	}
 }
 
+
 func (l *Linter) Lint(ctx context.Context, m *manifest.Manifest) (Results, error) {
 	// TODO: perform any porter level linting
 	// e.g. metadata, credentials, properties, outputs, dependencies, etc
+	reservedPrefixes := []string{"porter-", "porter_"}
```

Comment:
Let's change the comment above to remove the TODO since you have implemented a lint check now for porter. Can you instead leave a short comment about what you are checking for (the reserved porter prefix on parameter names)?

This will make it easier for people to figure out what we are checking for as we add more lint checks over time.